### ISSUE TITLE
Add the same buttons on the Tournesol recommandations as on the default recommandations: watch later etc

### ISSUE THREAD

**Issue by EtoiledeScauchy** [tournesol-app/tournesol-chrome-extension#39]
![image](https://user-images.githubusercontent.com/44090250/119242601-7c804380-bb5f-11eb-818c-179d4e94893c.png)

**Comment by lfaucon** [comment]
I wish Youtube's engineers would work on this! There is a chance that these options require some specific call to youtube's API that will be difficult for us to replicate. I also think that it is not extremely useful for us to add these options.

What wold you think of instead adding other options here:
- Thank the tournesol contributors (We have this Like option on Tournesol)
- Add to rate later
- Rate now
- See video details on Tournesol
- ...other ideas?

**Comment by EtoiledeScauchy** [comment]
I agree about the YouTube API issue, just in case I add it as future work. 

I'm less convinced about putting the Tournesol stuff, especially the Rate buttons on unopened videos seem like a bad incentive to rate without actually watching/paying attention.
